lib/matplotlib/patches.py

```diff
@@ -628,6 +628,12 @@ def __init__(self, patch, ox, oy, **kwargs):
         ox, oy : float
             The shift of the shadow in data coordinates, scaled by a factor
             of dpi/72.
+        shade : float, default: 0.3
+            How the darkness of the shadow relates to the original color. If 0, the
+            shadow is black, if 1, the shadow has the same color as the *patch*.
```

Comment:
I think you need to reverse numbers to have a reasonable semantics: High shade (=1) should be dark, low shade should be color; i.e.

default 0.7 and `color = (1- shade) * np.asarray(...)`

Alternatively we need a different name.

Not checked. Do we have something similar already in the library?

Reply:
I was also thinking about that, but couldn't really find something suitable. I think the `1 - shade`-solution is probably the easiest way out. I have not found anything similar at least, but may not really know what to search for.

https://en.wikipedia.org/wiki/Tint,_shade_and_tone#/media/File:Tint-tone-shade.svg

Reply:
I went with the "inverted shade" (so the version of shade that makes sense). I guess saturation will work as well, just that the shading is done in RGB-space so not clear which is better. But easy to change if required.

Reply:
Is everybody fine with the shading in RGB space or do we need to look into saturation?

Reply:
I guess it should ideally be done in some other space, but since we started using RGB (although with a fixed value), maybe we do not want to change it? Either way, it seems to be a slightly different problem.

Reply:
Maybe one should add a `color` argument to `Shadow` as well that takes precedence if given? In that way people can rely on `shade` if that gives good enough results or set whatever color they like and then benefit from the positioning etc.